### PR TITLE
feat: Add local count token example

### PR DESCRIPTION
**Pull request by msampathkumar**
## Description

Fixes # [b356425456](http://b/356425456)

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/count_token/count_token_example.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    response = model.count_tokens(prompt)
+    print(f"Prompt Token Count: {response.total_tokens}")
+    print(f"Prompt Character Count: {response.total_billable_characters}")
+
+    prompt = ["hello world", "what's the weather today"]
+    response = model.count_tokens(prompt)
+    print(f"Prompt Token Count: {response.total_tokens}")
+    print(f"Prompt Character Count: {response.total_billable_characters}")
+    # [END generativeaionvertexai_token_count_sample02]
+    return response.total_tokens
```
Comment (holtskinner) [inline generative_ai/count_token/count_token_example.py:59]: Is there a reason to have another sample for the service version? We already have https://github.com/GoogleCloudPlatform/python-docs-samples/blob/HEAD/generative_ai/gemini_count_token_example.py

Which can be edited to remove the `generate_content()` call.

**Review by holtskinner — APPROVED** [review]

File: generative_ai/count_token/count_token_example.py (hunk 1 above)
Comment (msampathkumar) [reply generative_ai/count_token/count_token_example.py:59]: Two reasons (1) To highlight the differences between local & SDK version (2) We are replacing the old sample with current one. (Removing old samples may happen at a later date)

Comment (msampathkumar) [reply generative_ai/count_token/count_token_example.py:59]: sure!

Comment (msampathkumar) [reply generative_ai/count_token/count_token_example.py]: PTAL @holtskinner & suggest.

**Pull request merged**